src/main/webapp/i18n/en/cleanupService.json

```
@@ -10,18 +10,54 @@
         "operation": {
             "deleteOrphans": "Delete orphans",
             "deletePlagiarismComparisons": "Delete plagiarism comparisons",
-            "deleteNonRatedResults": "Delete non-rated results",
-            "deleteOldRatedResults": "Delete old rated results",
+            "deleteNonRatedResults": "Delete feedback of old non-rated results",
+            "deleteOldRatedResults": "Delete feedback of old rated results",
+            "deleteOldSubmissionVersions": "Delete old submission versions",
+            "deleteOldFeedback": "Delete old feedback versions"
+        },
+        "tooltip": {
+            "deleteOrphans": "Deletes orphaned entities that are no longer associated with valid results or participations. This includes feedback, text blocks, and scores that reference null results, participations, or submissions",
+            "deletePlagiarismComparisons": "Deletes plagiarism comparisons with a status of \"None\" that belong to courses within the specified date range.",
```

Here we escape " and in the german translation we use ', we might want to be consistent there
```suggestion
            "deletePlagiarismComparisons": "Deletes plagiarism comparisons with a status of \"None\" that belong to courses within the specified date range.",
```